src/internal.c

```diff
@@ -14639,7 +14806,7 @@ static word32 QSH_KeyExchangeWrite(WOLFSSL* ssl, byte isServer)
                         return NO_PEER_KEY;
                     }
 
-                    wc_ecc_init(&myKey);
+                    wc_ecc_init_h(&myKey, ssl->heap);
```

Comment:
The naming scheme in ecc could use review. Appended _h to signify addition for heap usage but could be changed to _ex.